sqlglot/parser.py

```diff
@@ -950,6 +951,30 @@ def _retreat(self, index: int) -> None:
     def _parse_command(self) -> exp.Expression:
         return self.expression(exp.Command, this=self._prev.text, expression=self._parse_string())
 
+    def _parse_comment(self, allow_exists=True) -> exp.Expression:
+        start = self._prev
+        exists = self._parse_exists() if allow_exists else False
+
+        if not self._match(TokenType.ON):
```

Comment:
remove this and go back to COMMENT ON in the tokenizer

Reply:
will that work with the `COMMENT [IF EXISTS] ON` [snowflake syntax](https://docs.snowflake.com/en/sql-reference/sql/comment)?